core/src/main/java/bio/terra/pearl/core/service/kit/pepper/PepperKitRequest.java

```diff
@@ -0,0 +1,31 @@
+package bio.terra.pearl.core.service.kit.pepper;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import lombok.*;
+import lombok.experimental.SuperBuilder;
+
+import java.util.Set;
+
+/** holds kit request data as it is received from Pepper */
+@Getter @Setter
+@SuperBuilder @NoArgsConstructor
+@EqualsAndHashCode
+@ToString
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class PepperKitRequest {
```

Comment:
I think capturing that this is status or a response in the class name is important. The default reading of `PepperKitRequest` is as the payload for a Pepper request.

Reply:
or just PepperKit?

Reply:
I had not thought of that, but I think `PepperKit` works.